src/core/exchange/exchange-selectors.js

```diff
@@ -102,6 +102,16 @@ export function getExchangeRate(
 
   // Only search if the endpoints exist:
   if (search.routes[fromCurrency] && search.routes[toCurrency]) {
+    // Make sure that routes eventually point to a fiat currency
+    if (!/^iso:/.test(fromCurrency)) {
+      for (const fromRoute in search.routes[fromCurrency]) {
+        for (const deepRoute in search.routes[fromRoute]) {
```

Comment:
This second loop seems to have no effect? It just loops more or less times, and then we move to the add hint line in any case.

Suggestion: move the two loops into a function that returns a boolean, and base `addHint` on the result of calling the function. Then you can bail out of the middle loop with a `return true` (or false?) statement.